worlds/adult.wbt

```diff
@@ -1,74 +1,108 @@
-#VRML_SIM R2021a utf8
+#VRML_SIM R2021b utf8
 WorldInfo {
   info [
     "Description: official soccer simulation for the 2021 Robocup Virtual Humanoid League (adult size)"
-    "Version 0.1"
+    "Version 0.2"
   ]
   title "Robocup V-HL Adult"
-  basicTimeStep 16
+  basicTimeStep 8
```

Comment:
What's the benefit of changing the timestep? Or is that just the standard for robocup?

Reply:
Physics will update faster. I think it is the standard, or at least the conditions we'll be playing under (I don't think we're meant to get anymore field changes). It's not that necessary but may be useful if we want to play under their conditions.

Reply:
Fair enough, a smaller timestep will be great for vision data collection anyway so